Example fails with ng 12

Steps:

```
npm install --save angular-split material-design-icons typeface-open-sans typeface-roboto @angular-material-components/datetime-picker
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: ng@0.0.0
npm ERR! Found: @angular/cdk@12.2.13
npm ERR! node_modules/@angular/cdk
npm ERR!   @angular/cdk@"^12.2.13" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/cdk@"^13.0.1" from @angular-material-components/datetime-picker@7.0.1
npm ERR! node_modules/@angular-material-components/datetime-picker
npm ERR!   @angular-material-components/datetime-picker@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/thomaspeugeot/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/thomaspeugeot/.npm/_logs/2022-01-05T13_08_14_204Z-debug.log
````

It seems 7.0.1 from datetime-picker requests 13.0.1

--- a/README.md
+++ b/README.md
@@ -60,7 +60,7 @@ The four step is to define your front-end dependency by using the `tsconfig.json
 
 ## Gong is intended for system engineering tooling
 
-Gong fullstack approach was inspired by [Ruby on Rails](https://rubyonrails.org/) and the [conceptual compression concept](https://m.signalvnoise.com/conceptual-compression-means-beginners-dont-need-to-know-sql-hallelujah/).
+Gong fullstack approach was inspired by [Ruby on Rails](https://rubyonrails.org/) and a more generaly the idea that complexity facing the programmer should be carefuly managed, as it is described in [conceptual compression concept](https://m.signalvnoise.com/conceptual-compression-means-beginners-dont-need-to-know-sql-hallelujah/) and [Rob Pike's design of Go regarding complexity](https://www.dotconferences.com/2015/11/rob-pike-simplicity-is-complicated).
 
 Gong fullstack approach, with a backend in go, is similar in intent to [lorca](https://github.com/zserge/lorca), [wails](https://github.com/wailsapp/wails) and [fyne](https://github.com/fyne-io/fyne). However, the gong framework approach is different because it includes gongc, a go data model compiler to generate front-end and back-end code. In this sense, it is similar to [ent](https://github.com/ent/ent) which includes a ("shema as code") approach.
 

--- a/go/cmd/gongc/main.go
+++ b/go/cmd/gongc/main.go
@@ -274,7 +274,7 @@ func main() {
 
 				{
 					cmd := exec.Command("npm", "install", "--save",
-						"angular-split", "material-design-icons", "typeface-open-sans", "typeface-roboto", "@angular-material-components/datetime-picker")
+						"angular-split", "material-design-icons", "typeface-open-sans", "typeface-roboto", "@angular-material-components/datetime-picker@6.0.3")
 					cmd.Dir = gong_models.NgWorkspacePath
 					log.Printf("Installing some packages\n")
 

